src/components/input-field/input-field.tsx

```diff
@@ -408,46 +421,48 @@ export class InputField {
         );
     }
 
-    private renderDropdown() {
-        const filteredCompletions = this.filterCompletions(this.value);
+    private onKeyDown() {
+        this.showCompletions = true;
```

Comment:
currently you have to tab to select the list first before you can use the arrow key. wouldn't it be more natural to select the first item already by pressing `DOWN`? also if you tab twice the next input gets selected but the list stays open. check the `handleInputKeyDown` method in the picker, I think we need something similar here

Reply:
The behaviour for this should match the behaviour for similar elements.

I was going to write "like limel-menu" and then realised that limel-menu doesn't close if you tab away from it while it's open 😞

Then I checked limel-select, and that works pretty well, except that it loses focus when tabbing out of it (at least sometimes), so the focus ends up in the address bar instead of on the next element.

But, generally, we don't want to open the menu by tabbing, because that will make it a horrible experience to tab past a number of these elements. Instead, `arrow-up` and `arrow-down` should typically open the menu, and probably `enter` should too. Tab should tab away from the element, and if the menu is open when tabbing away, the menu should close.

Reply:
Note also that on macOS, `arrow-up` and `arrow-down` should normally place the caret (the text input marker) at the beginning and end of the text, respectively. So we might need to consider how that works together with the menu. We probably don't want to prevent that default behaviour, at least not when the menu isn't open… that's less important than getting the tabbing behaviour right though 🙂